02_bubble/bubble_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
Please add `TestBubbleSort` comparing returned []int slice to expected int slice.   
Include corner cases such as empty slice, one element slice, and duplicated elements.

Reply:
Done